packages/nimble-components/src/anchor/specs/README.md

```diff
@@ -0,0 +1,49 @@
+# Nimble Anchor
+
+## Overview
+
+The Nimble Anchor is a component used to navigate to a web resource, similar to the HTML anchor element.
+
+### Background
+
+[Visual Design spec](https://xd.adobe.com/view/33ffad4a-eb2c-4241-b8c5-ebfff1faf6f6-66ac/screen/bfadf499-caf5-4ca0-9814-e777fbae0d46)<br>
+[Nimble work item](https://github.com/ni/nimble/issues/324)
+
+---
+
+## Design
+
+### API
+
+[FAST's API documentation](https://github.com/microsoft/fast/blob/e576aa70c22780fffba03097277e2db9a2ec1cd8/packages/web-components/fast-foundation/src/anchor/README.md)
+
+-   _Component Name_: `nimble-anchor`
+-   _Properties/Attributes_: We will have two new properties/attributes: `underline` and `emphasis`. When present, the `underline` attribute will cause an underline to always be visible (as opposed to only on hover). Setting `emphasis` on the anchor will cause it to be shown in an accent color (as opposed to the text color). Furthermore, there will be a `disabled` property/attribute to make the anchor inoperable.
+-   _Methods_: Unchanged (none)
+-   _Events_: Unchanged (none)
+-   _CSS Classes and Custom Properties that affect the component_: Unchanged (none)
+-   _Slots_: We will not expose the start or end slot, because there isn't currently any need to.
+
+### Angular integration
+
+A directive will be created for the anchor component. The anchor does not participate in forms, so there will be no ControlValueAccessor.
```

Comment:
I'm not sure what, if anything, would need to be done for Blazor. It might be an area of investigation. I'll note that the `BreadcrumbItem` has no interesting Blazor implementation as it does for Angular.

Reply:
For Angular, yes, we need to specially support RouterLink. We'll follow the precedent set by the BreadcrumbItem, and I've updated the spec with details (mostly stolen from the Breadcrumb spec).

For Blazor, based on the Breadcrumb's wrapper, there's nothing additional to support.